docs/spec_draft.md

```diff
@@ -82,6 +82,17 @@ The specification of an op comprises of the following components (in the order
   * **Constraints** Type constraints on the operand(s), result(s).
   * **Examples** Examples demonstrating the working of the op.
 
+
+## Index of documented ops
+   * [AddOp](#stablehloadd)
```

Comment:
Let's use the lowercase names introduced later in the document, e.g. `add`, `constant`, etc? I don't think names like `AddOp` are used anywhere else.